### PR TITLE
Bugfix/star card

### PR DESCRIPTION
**Pull request by holladayian**
# Iteration Number
4
- [x] Are there any comments that need to be removed?
There are not!
- [x] Did you check all of the functionality of the webpage?
Yes
- [x] Do all parties understand every line of code?
Yes!
- [x] Is the code DRY/SRP
We did refactor as much as we could in the time limit. We were able to DRY up and create SRP for the things we could catch in this time.

## Questions
N/A

## Notes
We never got the class methods to update the local storage, but we ended up using a way that didn't need them.
We did find a possible bug with the search. It works, there is nothing that is broken, but we sure which way to go based on the comp.

## Problems
N/A

## Screenshot
N/A

@holladayian
@timkeresey
@saraho1123

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,6 +7,8 @@ Credit all teammates
 
 ### The Link to our github pages can be found here
 
+[link to GitHub Pages](http://timkeresey.github.io/ideabox-boilerplate/)
```
Comment (saraho1123) [inline README.md:10]: I think Tim is updating this from settings.

File: src/main.js [hunk 2]
```diff
@@ -120,74 +122,82 @@ function htmlInjector(index) {
...
       updateLocalStorage();
-    };
-  };
-  if (showAllButton.classList.contains('hidden')) {
+    }
+  }
+  checkView();
+}
+
+function checkView() {
```
Comment (saraho1123) [inline src/main.js:168]: This was a brilliant addition!

Comment (timkeresey) [inline src/main.js:168]: This is a good way to fix the bug.

**Review by saraho1123 — APPROVED** [review]
Great work team! It has been a pleasure to work with both of you. We seemed to have a great balance of challenging and uplifting each other and succeeded in accomplishing a lot with our code as well.

File: src/idea.js [hunk 3]
```diff
@@ -5,13 +5,15 @@ class Idea {
...
     this.star = star || false;
   }
+
   saveToStorage() {
-    var stringifiedArray = JSON.stringify(ideaArray);
-    localStorage.setItem("storedIdeas", stringifiedArray);
+
   }
+
   deleteFromStorage() {
```
Comment (timkeresey) [inline src/idea.js:13]: I'm not sure how we can incorporate this. We tried refactoring but kept breaking other things.

**Comment by timkeresey** [comment]
Great! It's too bad we didn't have time to fix some bugs and issues with the stylesheet. I dove into the CSS hard but couldn't resolve some problems. With more time the sidebar menu could have been refactored to work better, but I'm very happy with what we got done and how we worked as a team.

**Pull request merged**